src/constants/translations/en/cooperations-page.json

```diff
@@ -46,7 +46,8 @@
     "lessons": "lessons ",
     "or": "or ",
     "quizzes": "quizzes ",
-    "resourcesLibrary": "from resource library."
+    "resourcesLibrary": "from resource library.",
+    "studentEmptyCooperation": "Nothing in here. Please wait until tutor add some resources for this course. You can leave a message for the tutor in the "
```

Comment:
Seems like text inside translation is not corresponded to the [mockup](https://www.figma.com/file/liJZDYhUnDP15Pi9bipDcv/Space2Study-(Students)?type=design&node-id=17472-90474&mode=design&t=lWjsXCIUFXskHxAC-0).

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/a755c4c0-31f8-43b2-bf85-f425f6fd3097)

Reply:
Didn't know there was a mockup for it, thank you, solved